### PR TITLE
Make demo more appealing

### PR DESCRIPTION
**Pull request by blasten**
screen shot 2015-05-29 at 12 22 15 am

### CODE REVIEW

File: demo/x-app.html [hunk 1]
```diff
@@ -16,19 +16,23 @@
...
       @apply(--layout-horizontal);
+      @apply(--paper-font-body2);
     }
 
     [nav] {
-      width: 300px;
       @apply(--layout-vertical);
-      background-color: #ddd;
+
+      width: 300px;
```
Comment (notwaldorf) [inline demo/x-app.html]: Does this look ok on mobile? maybe you want `max-width` (I have learnt this the hard way)

File: demo/x-app.html [hunk 2]
```diff
@@ -37,31 +41,38 @@
...
     }
 
     [card] {
+      @apply(--layout-vertical);
+      @apply(--layout-center-center);
+
+      @apply(--shadow-elevation-2dp);
+
       height: 300px;
       width: 300px;
```
Comment (notwaldorf) [inline demo/x-app.html]: same comment about `max-width` here

Comment (blasten) [inline demo/x-app.html]: I agree here

File: demo/x-app.html (hunk 1 above)
Comment (blasten) [inline demo/x-app.html]: This is the main navigation, and it's usually of fixed width. However, I reduce the width to 200px.

**Comment by blasten** [comment]
PTAL? @notwaldorf

**Pull request merged**